modules/apps/headless/headless-portal-instances/headless-portal-instances-impl/rest-openapi.yaml

```diff
@@ -0,0 +1,204 @@
+components:
+    schemas:
+        PortalInstance:
+            # @review
+            properties:
+                active:
+                    readOnly: true
+                    type: boolean
+                companyId:
+                    format: int64
+                    type: integer
+                domain:
+                    description:
+                        domain used for email authentication.
+                    type: string
+                virtualhost:
+                    type: string
+                webId:
+                    type: string
+            type: object
+info:
+    description:
+        "A Java client JAR is available for use with the group ID 'com.liferay', artifact ID
+        'com.liferay.headless.admin.taxonomy.client', and version '2.0.1'."
```

Comment:
this should be empty and regenerate with headless-portal-instance

Reply:
ok @nhpatt, I will change it